test-requirements.txt

```diff
@@ -1,2 +1,3 @@
 factory-boy==2.1.1
 mock==2.0.0
+bs4
```

Comment:
looks like the correct name for this package is beautifulsoup4

https://pypi.org/project/bs4/